.github/workflows/label-assign.yml

```diff
@@ -0,0 +1,23 @@
+name: Label & Assign
+
+on:
+  issue_comment:
+    types: [created, edited]
+#  pull_request_review:
```

Comment:
sadly this doesn't work at the moment since `pull_request_review` runs in the context of the PR so doesn't have permissions to update labels or assignees.

I guess we need to wait for `pull_request_review_target` or something, see [here](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_review)

Reply:
> @PrettyWood I hope you don't mind me adding you here
Not at all, it won't change anything for me

> `pull_request_review` runs in the context of the PR
Oh that's a bummer...let's hope Github adds this soon